setup.py

```diff
@@ -1,6 +1,12 @@
 # coding: utf-8
 import sys
 from setuptools import setup, Command, find_packages
+try:
+    from pypandoc import convert
+    read_md = lambda f: convert(f, 'rst')
+except ImportError:
+    print("warning: pypandoc module not found, could not convert Markdown to RST")
```

Comment:
Print will not respect the logging formation of the platform.
Instead of `print`, `logging.error` or `logging.debug` can be used, which will reflect any runtime change regarding to logging formation.